tools/slang-capability-generator/capability-generator-main.cpp

```diff
@@ -78,85 +104,68 @@ struct CapabilityDef : public RefObject
         return expr.conjunctions[0].atoms[0];
     }
 
-    const HashSet<CapabilityDef*>& getAbstractAtomsPresent() const
-    {
-        return abstractAtomsPresent;
-    }
     void fillAbstractAtomsPresentFromCannonicalRepresentation()
     {
-        bool alreadySetTarget = false;
-        HashSet<CapabilityDef*> firstTargetSet;
-
-        bool alreadySetStage = false;
-        HashSet<CapabilityDef*> firstStageSet;
-
-        HashSet<CapabilityDef*> abstractFound;
-
-        for (auto& c : canonicalRepresentation)
+        HashSet<const CapabilityDef*> targetAbstractAtomsOfType;
```

Comment:
I find the this name confusing, what does `targetAbstractAtomsOfType` mean? Is it a set of all concrete target capDefs?

Reply:
I agree the name is confusing, it does not explain what is happening. 
The variable and related will be renamed to `sharedAbstractAtomsInCanonicalSet_target`, where the variable tracks the "shared abstract atoms in a canonical set for the 'target' atom type"